quail/utils.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from rpy2.robjects.packages import isinstalled, importr
+from pywps.app.exceptions import ProcessError
+
+
+logger = logging.getLogger("PYWPS")
+logger.setLevel(logging.NOTSET)
+
+formatter = logging.Formatter(
+    "%(asctime)s %(levelname)s: quail: %(message)s", "%Y-%m-%d %H:%M:%S"
+)
+handler = logging.StreamHandler()
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+
+def get_package(package):
+    if isinstalled(package):
+        return importr(package)
+    else:
+        raise ProcessError(f"R package, {package}, is not installed")
```

Comment:
I think this is something we would be able to share with `chickadee` no?

Reply:
Yes this is copy and pasted from chickadee, should be moved to wps-tools